playbooks/roles/minishift/tasks/main.yml

```diff
@@ -28,6 +28,10 @@
 - name: "Set minishift openshift-version to {{ oc_version }}"
   shell: "{{ minishift_bin }} config set openshift-version {{ oc_version }}"
 
+- name: "Set minishift minishift_insecure_registry"
+  shell: "{{ minishift_bin }} config set insecure-registry \"{{ minishift_insecure_registry }}\""
```

Comment:
does this take a list?

Reply:
yes

```
[root@host~]# minishift config view
- cpus                               : 2
- disk-size                          : 40gb
- memory                             : 6400mb
- openshift-version                  : v3.9.0
- skip-check-openshift-release       : true
- skip-check-openshift-version       : true

[root@host ~]# minishift config set insecure-registry registry1.example.com
[root@host ~]# minishift config view
- cpus                               : 2
- disk-size                          : 40gb
- insecure-registry                  : [registry1.example.com]
- memory                             : 6400mb
- openshift-version                  : v3.9.0
- skip-check-openshift-release       : true
- skip-check-openshift-version       : true

[root@host ~]# minishift config set insecure-registry registry1.example.com,registry2.example.com
[root@host ~]# minishift config view
- cpus                               : 2
- disk-size                          : 40gb
- insecure-registry                  : [registry1.example.com registry2.example.com]
- memory                             : 6400mb
- openshift-version                  : v3.9.0
- skip-check-openshift-release       : true
- skip-check-openshift-version       : true
```

Reply:
I think names should match to what minishift uses so should be singular IMHO